library/modules/Items.cpp

```diff
@@ -2133,14 +2133,17 @@ int32_t Items::createItem(df::item_type item_type, int16_t item_subtype, int16_t
     delete prod;
 
     DEBUG(items).print("produced %zd items\n", out_items.size());
-    if ( out_items.size() != 1 )
-        return -1;
 
     for (size_t a = 0; a < out_items.size(); a++ ) {
+        // Plant growths need a valid "growth print", otherwise they behave oddly
+        auto growth = virtual_cast<df::item_plant_growthst>(out_items[a]);
+        if (growth)
+            growth->growth_print = growth_print;
+
         out_items[a]->moveToGround(unit->pos.x, unit->pos.y, unit->pos.z);
```

Comment:
should probably use the `pos` variable calculated above, just in case the `unit->pos` values are invalid